tests/integration/targets/s3/aliases

```diff
@@ -1,2 +1,4 @@
 cloud/aws
 zuul/aws/cloud_control
+
+disabled  # Temporary disabled because they throw several ThrottlingExceptions and it needs further investigation
```

Comment:
Can you please make sure we have a ticket to remind us to come back to this?

Reply:
I created an issue here https://github.com/ansible-collections/amazon.cloud/issues/42. We do not have the jira label in this repo. If necessary, I can manually create a Jira ticket as well.